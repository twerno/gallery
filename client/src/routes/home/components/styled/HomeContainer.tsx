import styled from 'styled-components';

export default styled.div`
    display: flex;
    position: relative;
    height: 80vh;
    width: 100%;

    justify-content: center;
    align-items: center;
    flex-direction: column;
`;

