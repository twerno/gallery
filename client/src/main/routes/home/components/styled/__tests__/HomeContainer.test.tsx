import * as React from 'react';
import { componentBasicTests } from "test/componentBasicTests";
import HomeContainer from '../HomeContainer';

describe("HomeContainer", () => {

    componentBasicTests("HomeContainer", <HomeContainer />);

});