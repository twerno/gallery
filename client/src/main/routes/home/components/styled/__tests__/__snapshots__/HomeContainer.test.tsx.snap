// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContainer [HomeContainer] Component basic tests matches the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
  height: 80vh;
  width: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;
